projects/packages/password-checker/src/class-password-checker.php

```diff
@@ -348,29 +348,39 @@ protected function get_common_passwords() {
 	 * @return array user data.
 	 */
 	protected function get_other_user_data() {
-
 		if ( ! isset( $this->user ) ) {
 			$user_data = get_userdata( $this->user_id );
+		} else {
```

Comment:
@briancolinger I'm wondering if this is the desired approach.

when $this->user is not set than $user_data will be undefined. Instead of setting user_data as $this->user should we instead return with an empty array?

I do like the addition of isset around using of ID, user_email to ensure that data is set when those are used later on in the function.

Reply:
Fixed!